fusion-plugin-rpc/src/server.js

```diff
@@ -140,8 +142,30 @@ const pluginFactory: () => RPCPluginType = () =>
           const pathMatch = new RegExp(`${apiPath}([^/]+)`, 'i');
           const [, method] = ctx.path.match(pathMatch) || [];
           if (hasHandler(handlers, method)) {
+            let body;
             try {
-              await parseBody(ctx, () => Promise.resolve());
+              if (
+                ctx.req &&
```

Comment:
ctx.req should be always available

Reply:
yes ctx.req will be present but it is done to pass failing test cases. In some test cases which were already there req is missing.